docs/platform/atoms/booker.mdx

```diff
@@ -24,6 +24,66 @@ export default function Booker( props : BookerProps ) {
 }
 ```
 
+## Dynamic Bookings
+
+Dynamic bookings allow you to show combined availability across multiple users. This is useful for scenarios like:
+- Group meetings where all participants must be available
+- Sales calls that require multiple team members
+- Consultations with multiple experts
+
+### Using Dynamic Bookings
+
+Pass an array of usernames to create a dynamic booking:
+
+```js
+import { Booker } from "@calcom/atoms";
+
+export default function DynamicBooker() {
+  return (
+    <Booker
+      // Pass multiple usernames for combined availability
+      username={["alice", "bob", "charlie"]}
+      eventSlug="group-meeting"
+      onCreateBookingSuccess={() => {
+        console.log("Dynamic booking created!");
+      }}
+    />
+  )
+}
+```
+
+<Note>
+**Requirements for Dynamic Bookings:**
+- All specified users must have `allowDynamicBooking` enabled in their settings
```

Comment:
This is good information to keep in documentation . However, we need to mention that it is enabled by default as true so it's something to be aware only if dynamic bookings are not working.